contracts/facets/EmbalmerFacet.sol

```diff
@@ -153,7 +164,7 @@ contract EmbalmerFacet {
         );
 
         // Transfer the total fees amount in sarco token from the msg.sender to this contract
-        sarcoToken.transferFrom(msg.sender, address(this), totalFees);
+        s.sarcoToken.transferFrom(msg.sender, address(this), totalFees);
 
         // Emit the event
         emit LibEvents.InitializeSarcophagus(
```

Comment:
Can `minShards` be included in the emitted event?

Reply:
Will do.

Reply:
Scratch that. Getting the stack too deep error again. I'll have to think of another way.

Reply:
@knoll3 -- dont worry about making this update --- we can get it off the contract data if we need to